client/app/dbaas/dbaas-metrics/dashboard/metrics-dashboard.controller.js

```diff
@@ -98,6 +98,11 @@
                     text: this.$translate.instant("common_manage"),
                     href: this.ControllerHelper.navigation.getUrl("contacts", { serviceName: this.serviceName }),
                     isAvailable: () => true
+                },
+                editName: {
+                    text: this.$translate.instant("metrics_tiles_modify"),
```

Comment:
One day we should update translations with common ones, I think metrics_tiles_modify equals to common_edit